### ISSUE TITLE
Compile to ES5

### ISSUE THREAD

**Issue by jwworth** [perogi/zipcodes-perogi#34]
Thanks for creating this; it's really cool!

We're trying to use this library with create-react-app, which requires that dependencies be compiled to ES5 as a build step:

https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#npm-run-build-fails-to-minify

Would it be possible to publish the library pre-compiled to ES5? Thanks in advance if you have the time and interest in making this change.

**Comment by perogi** [comment]
Hi @jwworth,

Thank you for your interest!

From your link it looks like upgrading to react-scripts@2.0.0 or above will fix this issue for your team.  Funny that you post this, at my workplace we had exactly the same issue and upgraded to the latest react-scripts and everything worked fine.  We eventually ejected react-scripts so we had more control over the project (I have to say ejecting was a bit of a pain but we got it working).

I'm not totally against precompiling it to ES5, I only want to try to keep the project as simplistic as possible on my end.

Thanks again and have a great day,
perogi.
